jib-core/src/main/java/com/google/cloud/tools/jib/cache/LayerMetadata.java

```diff
@@ -31,23 +31,23 @@
   static class LayerMetadataEntry {
 
     /** The source file path string, in OS-specific format. */
```

Comment:
Just a question: so this path will contain "/" on Unix and "\" on Windows, for example?

Reply:
Ah nope, need to update this comment.